cliconfig/file.go

```diff
@@ -48,7 +49,7 @@ func (f *File) Load() error {
 		if !isIgnoredLine(fullLine) {
 			key, value, err := parseLine(fullLine)
 			if err != nil {
-				return err
+				return fmt.Errorf("parsing config line: %w", err)
```

Comment:
Should it include the line number?

Reply:
ooh that's a good idea! yes, will implement it